scripts/docker_image/docker_image_entry_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "WARNING: This Docker image will be removed permanently on 2024/12/1. Please STOP using this image ASAP"
```

Comment:
Move to be with legacy Dockerfile

Reply:
not moved, just needed to delete from here. Done in [439e46e](https://github.com/dnanexus/dxCompiler/pull/481/commits/439e46e14198b41b361e2ed325662acef845179c)